lib/textstat.rb

```diff
@@ -23,7 +23,7 @@ def self.syllable_count(text, language = 'en_us')
     count = 0
     text.split(' ').each do |word|
       word_hyphenated = dictionary.visualise(word)
-      count += [1, word_hyphenated.count('-') + 1].max
```

Comment:
Not sure what the original intent was here. `count` will always return >= 0 and adding one will always be >= 1 so the `max` of that and one will always be >= 1?!